RELEASE_NOTES.md

```diff
@@ -1,7 +1,14 @@
 [//]: # (Consider renaming or removing the header for next release, otherwise it appears as duplicate in the published release, e.g: https://github.com/mongodb/mongodb-enterprise-kubernetes/releases/tag/1.22.0 )
 <!-- Next Release -->
+# MCK 1.0.1 Release Notes
 
-# MCK 1.0 Release Notes
+This is a patch release that addresses issue with OpenShift / OLM Operator bundle released in 1.0.0.
```

Comment:
I think this can be removed, we never added a summary at the top of a release. The mention below covers this already.

```suggestion
```